internal/connection_repository/sql_connection_locator.go

```diff
@@ -134,21 +134,28 @@ func (scm *SqlConnectionLocator) GetConnectionsByAccount(ctx context.Context, ac
 
 	for rows.Next() {
 		var client_id domain.ClientID
+		var orgIdString string
```

Comment:
For now, the type of the org_id passed to the `rows.Scan()` will need to be a `sql.NullString` because the org_id can be null in the database.